pkg/config/config.go

```diff
@@ -1229,7 +1229,9 @@ func InitConfig(config Config) {
 	setupProcesses(config)
 }
 
-var ddURLRegexp = regexp.MustCompile(`^app(\.(us|eu)\d)?\.(datad(oghq|0g)\.(com|eu)|ddog-gov\.com)$`)
+// ddURLRegexp determines if an URL belongs to Datadog or not. If the URL belongs to Datadog it's prefixed with the Agent
+// version (see AddAgentVersionToDomain).
+var ddURLRegexp = regexp.MustCompile(`^app(\.[a-z]{2}\d)?\.(datad(oghq|0g)\.(com|eu)|ddog-gov\.com)$`)
```

Comment:
Not a blocker but it would be nice to have these two regex strings next to each other in some package.

Reply:
I agree, I wanted to move them but I'm not sure either `config` nor `forwarder` make sens for this. I feel that we should have a `url`-something package dedicated to this. But with the `additional_endpoints` feature it make such package complex to introduce in the right places.

For another PR I think.